packages/appstore/src/components/modals/cashier-wallet-modal/provider.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+
+export const getCashierOptions = (is_demo: boolean) => {
```

Comment:
@heorhi-deriv This is not maintainable in the long term, As we can see there is already `is_visible` prop hard coded to `true` just because it has to match with other objects in the list. Let's just create components and render them in the parent 🙏🏻

Reply:
@farzin-deriv after double checking of design i found out that we should handle `demo|real|p2p|payment_agent` wallets with different amount of tabs with different components inside (deposit component in real and payment_agent are different components), so i refactored provider. now visibility is handled by the type of wallet.